doc/wiki.txt

```diff
@@ -167,29 +167,27 @@ GUIDE                                                        *wiki-intro-guide*
 
 Executive summary~
 * Speed is key - make sure you have a fast workflow.
-  * Use a global keyboard shortcut to open the wiki index, e.g. <alt-n>.
+  * Use a global keyboard shortcut to map |:WikiIndex| to open the wiki index,
```

Comment:
No, that's not really what I mean. When I say _global_ I don't mean a Vim mapping, but a mapping at the window/desktop manager level. I believe this is more clear in the following text, but I can see that it is unclear here.

Reply:
Oops. Yeah, I never realized you are referring to the wm level, but it makes sense. Maybe it needs some clarification as "global" can be interpreted in many ways (I thought not at the buffer level)

Reverted with 5f386b4936120ac918b244bd3c42a74db61cde64